FreeRTOS/FreeRTOS-Partner-Supported-Demos
# Add demo project for RH850 CCRH U2x and GHS U2x

Add demo projects for RH850 CCRH U2x and GHS U2x

Description
-----------
This directory includes four demo projects for the RH850 U2x devices, utilizing the CC-RH and GHS compilers, and running the FreeRTOS kernel. The projects are as follows:

`RH850_U2Ax_CCRH`: A demo project for the `CCRH` compiler, supporting single and multicore configurations, tested on the `U2A8 device`.
`RH850_U2Ax_GHS`: A demo project for the `GHS` compiler, supporting single and multicore configurations, tested on the `U2A8 device`.
`RH850_U2Bx_CCRH`: A demo project for the `CCRH` compiler, supporting single and multicore configurations, tested on the `U2B6 device`.
`RH850_U2Bx_GHS`: A demo project for the `GHS` compiler, supporting single and multicore configurations, tested on the `U2B6 device`.

Test Steps
-----------
The testing is following the template in [Template ThirdParty](https://github.com/FreeRTOS/FreeRTOS/blob/main/FreeRTOS/Demo/ThirdParty/Template/README.md)

Related Issue
-----------
None

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
